### PR TITLE
add content-type.

### PR DESCRIPTION
**Pull request by disksing**
Some libs will check it, java-grpc for example.

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -83,6 +83,7 @@ impl GrpcClient {
             Header::new(":path", method.name.clone()),
             Header::new(":authority", host.clone()),
             Header::new(":scheme", http_scheme.as_bytes()),
+            Header::new("content-type", "application/grpc"),
```
Comment (siddontang) [inline src/client.rs:86]: the standard?

Comment (disksing) [reply src/client.rs:86]: Please refer to https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md

**Comment by siddontang** [comment]
LGTM

File: src/client.rs (hunk 1 above)
Comment (overvenus) [reply src/client.rs:86]: How about `application/grpc+proto`, since all messages are protobuf encoded.

Comment (disksing) [reply src/client.rs:86]: Not sure if it will support json later, btw the golang implementation is also always use "application/grpc".

**Pull request closed without merging**